config/core/build-configs-cip.yaml

```diff
@@ -237,6 +237,40 @@ build_configs:
               - 'allnoconfig'
               - 'cip://5.10.y-cip/riscv/qemu_riscv64_defconfig'
 
+  cip_6.1:
```

Comment:
Hello @aliceinwire 

I believe we need to move this below `cip_5.10-rt` to maintain alpha order.

Thanks, Chris

Reply:
this is just the pr for updating the isar-cip-core not 6.1. I used same branch for another pr and the commit come also here. by the way for replaying to your question, there is no alpha order, the order is checked (I think alphabetically) by githubCI and that's the only order that doesn't give an error on the PR